roles/variables/defaults/main.yml

```diff
@@ -212,9 +212,12 @@ ssl_enabled: false
 ### Set this variable to customize expiration days for certificate authority. Applies for all components of Confluent Platform.
 certificate_authority_expiration_days: 365
 
-### Boolean to enable mTLS Authentication on all components. Configures all components to use mTLS for authentication into Kafka
+### Boolean to enable mTLS Authentication on all components. Configures all components to use mTLS for authentication into Kafka. No need to define this anymore, use ssl_client_authentication instead.
```

Comment:
we can instead mark this variable as deprecated

Reply:
I had a little confusion there as internally at most places we do continue to use this variable. Just at the end when we need the value we use the new variable. Even in that case should we mark it deprecated ?